src/ReaderScreen/utils/util.js

```diff
@@ -1,25 +1,25 @@
 import React from "react";
 import { Text } from "react-native";
-import { baseFontSize } from "../../common/helpers";
+import baseFontSize from "../../common/helpers";
 import constant from "../../common/constant";
 import colors from "../../common/colors";
 import { styles } from "../styles/styles";
 
 export const fontColorForReader = (header, nightMode, text) => {
+  const { HEADER_COLOR_1_DARK, HEADER_COLOR_1_LIGHT, HEADER_COLOR_2_LIGHT, SLIDER_TRACK_MIN_TINT } =
+    colors;
   const colorMapping = {
     [constant.GURMUKHI]: {
-      1: nightMode ? colors.HEADER_COLOR_1_DARK : colors.HEADER_COLOR_1_LIGHT,
-      2: nightMode ? colors.SLIDER_TRACK_MIN_TINT : colors.HEADER_COLOR_2_LIGHT,
-      6: nightMode ? colors.SLIDER_TRACK_MIN_TINT : colors.HEADER_COLOR_2_LIGHT,
+      1: nightMode ? HEADER_COLOR_1_DARK : HEADER_COLOR_1_LIGHT,
+      2: nightMode ? SLIDER_TRACK_MIN_TINT : HEADER_COLOR_2_LIGHT,
+      6: nightMode ? SLIDER_TRACK_MIN_TINT : HEADER_COLOR_2_LIGHT,
+      default: nightMode ? SLIDER_TRACK_MIN_TINT : HEADER_COLOR_2_LIGHT,
     },
-    [constant.TRANSLITERATION]: nightMode
-      ? colors.HEADER_COLOR_1_DARK
-      : colors.HEADER_COLOR_1_LIGHT,
-    [constant.TRANSLATION]: nightMode ? colors.SLIDER_TRACK_MIN_TINT : colors.HEADER_COLOR_2_LIGHT,
+    [constant.TRANSLITERATION]: nightMode ? HEADER_COLOR_1_DARK : HEADER_COLOR_1_LIGHT,
+    [constant.TRANSLATION]: nightMode ? SLIDER_TRACK_MIN_TINT : HEADER_COLOR_2_LIGHT,
```

Comment:
Similarly here, instead of adding nightMode conditional in each key, can we move it one level up, and set values accordingly.

Reply:
fixed